python/lsst/fgcmcal/fgcmBuildStarsBase.py

```diff
@@ -616,8 +622,8 @@ def _fillVisitCatalog(self, visitCat, groupedDataRefs, bkgDataRefDict=None,
                                for bg in bgList)
                     rec['skyBackground'] = sum(np.median(bg[np.isfinite(bg)]) for bg in bgStats)
                 else:
-                    self.log.warn('Sky background not found for visit %d / ccd %d' %
-                                  (visit, det))
+                    self.log.warning('Sky background not found for visit %d / ccd %d' %
+                                     (visit, det))
```

Comment:
Whoops...this one ended up in the wrong commit 😉

Also, the commit summary line exceeds the 50 char limit (https://developer.lsst.io/work/flow.html#writing-commit-summary-lines).

Reply:
Yeah, I had noticed that this was on a different commit, but I didn't think that was important...also I fixed the summary line.